Fixed missing "CONFIG_" perfix to enable LWM2M_FIRMWARE_UPDATE_OBJ_SUPPORT_MULTIPLE

henbro12 opened the pull request:
Fixed missing '_CONFIG' prefix when enabling LWM2M_FIRMWARE_UPDATE_OBJ_SUPPORT_MULTIPLE and CONFIG_LWM2M_FIRMWARE_UPDATE_OBJ_INSTANCE_COUNT through prj.conf

Signed-off-by: henbro brom <henrico.brom@vention.nl>

rlubos reviewed (commented):
Looks good, but please follow the commit guidelines when it comes to commit message: https://docs.zephyrproject.org/latest/contribute/guidelines.html#commit-guidelines. See compliance results for details.

VT-Henbro committed:
net: lwm2m: Fixed missing _CONFIG prefix

Fixed missing '_CONFIG' prefix when enabling
LWM2M_FIRMWARE_UPDATE_OBJ_SUPPORT_MULTIPLE and
CONFIG_LWM2M_FIRMWARE_UPDATE_OBJ_INSTANCE_COUNT through prj.conf

Signed-off-by: Henrico Brom <henrico.brom@vention.nl>

henbro12 commented:
Sorry this is my first contribution and I am not authorized, @rlubos could you merge this PR?

rlubos commented:
> Sorry this is my first contribution and I am not authorized, @rlubos could you merge this PR?

No, I don't have merge permissions, it just needs to wait for a maintainer.
